<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CBIAttrDialog</class>
 <widget class="QDialog" name="CBIAttrDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="m_pFreeRadioButton">
       <property name="text">
        <string>自由组合</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_pRelyHardRadioButton">
       <property name="text">
        <string>保护硬开入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_pRlySoftRadioButton">
       <property name="text">
        <string>保护软遥信</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_pRelyGSRadioButton">
       <property name="text">
        <string>保护GS开入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_pCheckBIradioButton">
       <property name="text">
        <string>检修压板</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_pResetBIradioButton">
       <property name="text">
        <string>复归开入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_pRemoteOpradioButton">
       <property name="text">
        <string>远方操作硬压板</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="m_DB0CheckBox">
         <property name="text">
          <string>DB0:可见标志</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB1CheckBox">
         <property name="text">
          <string>DB1:信号上送标志</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB2CheckBox">
         <property name="text">
          <string>DB2:录波标志</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB3CheckBox">
         <property name="text">
          <string>DB3:保护开入</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB4CheckBox">
         <property name="text">
          <string>DB4:单点硬遥信</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB5CheckBox">
         <property name="text">
          <string>DB5:MNG单点软遥信</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB6CheckBox">
         <property name="text">
          <string>DB6:RLY单点软遥信</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB7CheckBox">
         <property name="text">
          <string>DB7:双点遥信</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB8CheckBox">
         <property name="text">
          <string>DB8:整型遥信</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB9CheckBox">
         <property name="text">
          <string>DB9:检修压板</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB10CheckBox">
         <property name="text">
          <string>DB10:复归开入</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB11CheckBox">
         <property name="text">
          <string>DB11:五防解锁</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB12CheckBox">
         <property name="text">
          <string>DB12:间隔五防</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB13CheckBox">
         <property name="text">
          <string>DB13:监控五防</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB14CheckBox">
         <property name="text">
          <string>DB14:远方操作硬压板</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB15CheckBox">
         <property name="text">
          <string>DB15:备用</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="m_DB16CheckBox">
         <property name="text">
          <string>DB16:备用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB17CheckBox">
         <property name="text">
          <string>DB17:备用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB18CheckBox">
         <property name="text">
          <string>DB18:备用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB19CheckBox">
         <property name="text">
          <string>DB19:备用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB20CheckBox">
         <property name="text">
          <string>DB20:备用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB21CheckBox">
         <property name="text">
          <string>DB21:备用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB22CheckBox">
         <property name="text">
          <string>DB22:备用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB23CheckBox">
         <property name="text">
          <string>DB23:备用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB24CheckBox">
         <property name="text">
          <string>DB24:备用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB25CheckBox">
         <property name="text">
          <string>DB25:备用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB26CheckBox">
         <property name="text">
          <string>DB26:备用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB27CheckBox">
         <property name="text">
          <string>DB27:备用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB28CheckBox">
         <property name="text">
          <string>DB28:备用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB29CheckBox">
         <property name="text">
          <string>DB29:备用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB30CheckBox">
         <property name="text">
          <string>DB30:备用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_DB31CheckBox">
         <property name="text">
          <string>DB31:单点GS开入</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CBIAttrDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CBIAttrDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
